pkg/config/user_config.go

```diff
@@ -646,13 +646,14 @@ type CustomCommand struct {
 	LoadingText string `yaml:"loadingText" jsonschema:"example=Loading..."`
 	// Label for the custom command when displayed in the keybindings menu
 	Description string `yaml:"description"`
-	// If true, stream the command's output to the Command Log panel
-	// [dev] Pointer to bool so that we can distinguish unset (nil) from false.
-	Stream *bool `yaml:"stream"`
-	// If true, show the command's output in a popup within Lazygit
-	// [dev] Pointer to bool so that we can distinguish unset (nil) from false.
-	ShowOutput *bool `yaml:"showOutput"`
-	// The title to display in the popup panel if showOutput is true. If left unset, the command will be used as the title.
+	// Where the output of the command should go. 'none' discards it, 'log' shows it in the command log, and 'popup' shows it in a popup.
```

Comment:
I think it's worth mentioning that 'log' will stream the output whereas 'popup' will only show the output after the command has finished

Reply:
Why? This feels like a matter of course to me. With 'log', we stream because it's easy to do and it would be kind of silly not to, but it isn't essential. And for 'popup' it makes little sense to stream the output to the popup window (although technically we might be able to). Why do you think it needs mentioning?

(Happy to include this if it doesn't make the text a lot longer. Would you say it's enough to replace "'log' shows it in the command log" with "'log' streams it to the command log"?)

Reply:
I just don't expect the user to have that level of understanding that we do about what makes sense. But yes I'm happy to say 'log streams it to the command log'

Reply:
Changed in the new version